### PR TITLE
Shweta trivia

### PR DESCRIPTION
**Pull request by ShwetaHere**
Complete 12 out of 14 test cases for Multiple Choice option test case.

### CODE REVIEW

File: src/test/java/definitions/TriviaShwetaStepdefs.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        String actualUrl = getDriver().getCurrentUrl();
+        assertThat(actualUrl.contains("/quiz-builder/"));
+
+    }
+
+    @Then("I should still be at {string} page")
+    public void iShouldStillBeAtPage(String expectedUrl) {
+        String actualUrl = getDriver().getCurrentUrl();
+        System.out.println( "actualUrl : " + actualUrl);
+        Assert.assertTrue(actualUrl.contains("expectedUrl"));
```
Comment (gpl-git) [inline src/test/java/definitions/TriviaShwetaStepdefs.java:245]: Assert.assertTrue(actualUrl.contains("expectedUrl"));
You should use variable name here
Assert.assertTrue(actualUrl.contains(expectedUrl));

**Review by gpl-git — COMMENTED** [review]
There are duplicated steps in your code. Please remove

**Comment by ShwetaHere** [comment]
Where are they Galina, Can you please point it out. I am not able to find.

File: src/test/java/definitions/TriviaShwetaStepdefs.java (hunk 1 above)
Comment (ShwetaHere) [reply src/test/java/definitions/TriviaShwetaStepdefs.java:245]: Oh, by mistake I think it happened.

**Comment by ShwetaHere** [comment]
Galina, the duplicate step is coming from different student's file. not mine step definition. What to do?

**Comment by ShwetaHere** [comment]
PassConfirmPassStepDefs.java
This is the file from where the definition is being detected as duplicated. 
I have changed my step definition. Hope it will work

**Pull request closed without merging**